app/services/recovery_code_generator.rb

```diff
@@ -1,7 +1,7 @@
 class RecoveryCodeGenerator
   attr_reader :user_access_key
 
-  def initialize(user, length: 5)
+  def initialize(user, length: 4)
```

Comment:
what's the thinking on changing this? don't see any mention in the PR description